README.rst

```diff
@@ -61,14 +61,16 @@ nise is a command line tool. Currently only accepting a limited number of argume
 
 - *--start-date YYYY-MM-dd* (not supplied, if using --static-report-file yaml)
 - *--end-date YYYY-MM-dd* (optional, defaults to today and current hour)
-- (--aws | --ocp) required provider type
+- (--aws | --ocp | --gcp) required provider type
 - *--aws-s3-bucket-name bucket_name*  (optional, must include --aws-s3-report-name) Note: Use local directory path to populate a "local S3 bucket".
 - *--aws-s3-report-name report_name*  (optional, must include --aws-s3-bucket-name)
 - *--aws-s3-report-prefix prefix_name*  (optional)
 - *--aws-finalize finalize_choice* (optional, choices: ['copy', 'overwrite'])
 - *--ocp-cluster-id cluster-id* (required when providing ocp type)
 - *--insights-upload UPLOAD_URL* (optional) Note: Use local directory path to populate a "local upload directory".
 - *--static-report-file file_name* (optional) Note: Static report generation based on specified yaml file.  See example_aws[ocp]_static_data.yml for examples.
+- *--gcp-report-prefix prefix_name*  (optional)
+- *--gcp-bucket-name bucket_name*  (optional, must have GOOGLE_APPLICATION_CREDENTIALS environment variable set)
```

Comment:
Maybe clarify that comment to indicate that if the value is a path on disk, files will be written locally. Only if it's not a path would it need the credentials env variable set.

Also, we should explain what kind of value is expected in `GOOGLE_APPLICATION_CREDENTIALS`.